dotCMS/src/main/java/com/dotmarketing/util/CompanyUtils.java

```diff
@@ -1,10 +1,12 @@
 package com.dotmarketing.util;
 
+import com.dotcms.business.CloseDBIfOpened;
 import com.dotmarketing.cms.factories.PublicCompanyFactory;
 import com.liferay.portal.model.Company;
 
 public class CompanyUtils {
 
+	@CloseDBIfOpened
 	public static Company getDefaultCompany()
 	{
 		Company company = PublicCompanyFactory.getDefaultCompany();

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "PublicCompanyFactory"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)